src/main/java/org/json/JSONTokener.java

```diff
@@ -517,11 +520,11 @@ public JSONException syntaxError(String message, Throwable causedBy) {
     /**
      * Make a printable string of this JSONTokener.
      *
-     * @return " at {index} [character {character} line {line}]"
+     * @return " at index {index} [character number {character} in line {line}]"
      */
     @Override
     public String toString() {
-        return " at " + this.index + " [character " + this.character + " line " +
```

Comment:
Please revert this change. While improving readability is good, I think the benefit is outweighed by the potential cost of broken unit tests by the library users.

Reply:
reverted.